src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,51 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private static final int AMOUNT_OF_OBJECTS = 2;
+    private static final int KEY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private int currentIndex = 0;
+
+    private Object[][] array = new Object[MAX_SIZE][AMOUNT_OF_OBJECTS];
```

Comment:
```suggestion
    private Object[][] array;
    
    public StorageImpl() {
        array = new Object[MAX_SIZE][AMOUNT_OF_OBJECTS];
    }
```

maybe it would be better to create array[amount_of_objects][max_size]. In this case you create only two array with 10 elements each. And in your case - 10 array with 2 elements. But its up to you and you could or could not change it =)